serverless.yml

```diff
@@ -55,6 +62,7 @@ functions:
     role: arn:aws:iam::${self:custom.accountNumber}:role/csr-Lambda-Role
     environment:
       AWS_DEPLOYMENT_REGION: ${self:provider.region}
+      LOG_LEVEL: INFO
```

Comment:
Look at this, it occurs to me that I don't think the `MAX_RETRIES` environment variable is actually used by the code. :thinking: